pkg/api/v1/types.go

```diff
@@ -2002,6 +2002,8 @@ type ListOptions struct {
 	// When specified with a watch call, shows changes that occur after that particular version of a resource.
 	// Defaults to changes from the beginning of history.
 	ResourceVersion string `json:"resourceVersion,omitempty"`
+	// FIXME:
+	TimeoutSeconds *int64
```

Comment:
We should probably use timeoutSeconds everywhere, but should switch to preferring it over the existing `timeout` supported by the apiserver and we would want to deprecate its use in favor of timeoutSeconds always.   Unless we think the two values (timeout of how long the server waits for the rest handler to respond, and timeout of how long the rest handler waits on the underlying db) are significantly different.